rpmconf/Install/Util/utilfunc.sh

```diff
@@ -1715,8 +1715,8 @@ removeExistingPackages() {
   fi
 }
 
-removeZextrasPackagesIfInstalled() {
```

Comment:
can we keep it as before and create new method to remove deprecated zimlets

Reply:
I thought it to keep a common method for removing not required packages.

Reply:
Done